### ISSUE TITLE
也是像原版那样使用吗

### ISSUE THREAD

**Issue by SilentTE** [viktorfa/oot_diffusion#3]
python main.py --model_path /root/OOTDiffusion/run/examples/model/01008_00.jpg --cloth_path /root/OOTDiffusion/run/examples/garment/049940_1.jpg --scale 2.0 --sample 4

**Comment by viktorfa** [comment]
No. You cannot run it from the command line now. See the readme or the code for how to create your own main function.
